zebra/kernel_netlink.c

```diff
@@ -308,17 +308,7 @@ static unsigned int netlink_file_counter = 1;
 static char netlink_fuzz_file[MAXPATHLEN] = "";
 
 /* Flag for whether to read from file or not */
-int netlink_read;
-
-/**
- * netlink_set_read() - Sets the read flag
- * @flag:      Flag value.
- *
- */
-void set_netlink_read(int flag)
-{
-	netlink_read = flag;
-}
+bool netlink_read = false;
```

Comment:
This is triggering a CI warning since a global in this case would be initialized to false/zero by default.